Ginger/GingerCore/Drivers/Appium/SeleniumAppiumDriver.cs

```diff
@@ -201,78 +224,103 @@ public bool ConnectToAppium()
         {
             try//Adding back the Try-Catch because without it in case of connection issue Ginger crash
             {
-                //Check if we need to use old IP and Port settings
-                if (String.IsNullOrEmpty(AppiumServer)==false) {
-                    //Get IP address and port from old settings - AppiumServer string value
-                    GetPortAndIPFromOldSettings();
+                Uri serverUri = null;
+                if (String.IsNullOrEmpty(AppiumServer) == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6113827)